go/vt/orchestrator/inst/instance_dao.go

```diff
@@ -376,10 +376,10 @@ func ReadTopologyInstanceBufferable(instanceKey *InstanceKey, bufferWrites bool,
 			go func() {
 				defer waitGroup.Done()
 				err := sqlutils.QueryRowsMap(db, "show global status like 'rpl_semi_sync_%'", func(m sqlutils.RowMap) error {
-					if m.GetString("Variable_name") == "Rpl_semi_sync_master_status" {
-						instance.SemiSyncMasterStatus = (m.GetString("Value") == "ON")
-					} else if m.GetString("Variable_name") == "Rpl_semi_sync_master_clients" {
-						instance.SemiSyncMasterClients = m.GetUint("Value")
+					if m.GetString("Variable_name") == "Rpl_semi_sync_primary_status" {
+						instance.SemiSyncPrimaryStatus = (m.GetString("Value") == "ON")
```

Comment:
wrong. MySQL variable name unaffected by this PR

Reply:
Fixed